.github/workflows/ruby.yml

```
@@ -7,22 +7,20 @@ on:
     branches: [ master ]
 
 jobs:
-  rails_3_2:
+  rails_6_1_7:
     runs-on: ubuntu-latest
 
     steps:
     - uses: actions/checkout@v2
```

actionlint (1.7.4)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)